### DISCUSSION TITLE
VERSION 1.O.2 Wcfm Vendor Direct Pay (Plugin) - compatibility

### DISCUSSION THREAD

**Discussion by Yan-Aleman** [inspireui/support#5351] [New Feature/Ideas Requests] [answered]
WFCM has released a very important addon that allows Vendors to choose the preferred payment method for customers in "personal account Vendors"

______

The developers have added an update to the plugin.  Also in the future, they will add it at the system level to the main WCFM plan.

New VERSION 1.0.2
https://drive.google.com/file/d/1qGLSRccT5qZHadJ9SGTmteirhbj7O7Mv/view?usp=sharing

______

Info here:
https://wordpress.org/support/topic/payment-gateways-by-vendor/#post-12880625

Please make this feature compatible by default for Fluxstore Multi. It is very important and solves many problems. Since not all Vendors want to receive payment by card or cash.
_____

Also "Payment Details" allows Vendors to attach text to the payment description. And the buyer after placing an order sees this text. Very handy when the payment method selected is bank transfer and Vendors can enter their account number in the Payment Details. The buyer will simply see it and make a transfer.

**Comment by nikhilbhoyar** [comment, marked as answer]
This is indeed a important feature.

**Comment by gregkeys** [comment]
Vendors should be able to setup how they want to receive payment as well as how they want the customers to pay from within the app

**Comment by ameenbadri7** [comment]
nice feature.
we need it.

**Comment by Yan-Aleman** [comment]
Also "Payment Details" allows Vendors to attach text to the payment description. And the buyer after placing an order sees this text. Very handy when the payment method selected is bank transfer and Vendors can enter their account number in the Payment Details. The buyer will simply see it and make a transfer.
![Снимок экрана 2022-11-07 205258](https://user-images.githubusercontent.com/36808995/200342934-708bc40b-68a4-4a08-a1bc-62fcd8934368.jpg)

**Comment by Yan-Aleman** [comment]
@minhcasi vital function👆🙏

**Comment by ecomdeveloperbd** [comment]
it's very important

**Comment by AkimotoKazue** [comment]
The best feature. You helped me a lot. Now the Vendors are happy. In Payments Details, vendors can now leave a thank you note or bank transfer details, or disable this payment method. It would also be cool to see this feature in the Fluxstore MultiVendors app.

**Comment by Yan-Aleman** [comment]
@simonpham

**Comment by ferchuchin** [comment]
Hi, I know this is an old post, but... Is it posible to make it  HPOS compatible?
![Screenshot 2023-11-27 135100](https://github.com/inspireui/support/assets/65634566/3b5cbe51-bb4e-4600-ae6c-8ee482fc127c)

**Reply by Yan-Aleman** [reply]
Hello.  Did this plugin work in the fluxstore application before hpos was included?  when choosing a payment method for your order!

**Reply by ferchuchin** [reply]
Hi.. I just use it  on a wcfm marketplace website... I don't know what
fluxstore is...but this is the only plugin stopping the HPOS migration

**Comment by EthanWilsondev** [comment]
A primary function, because some vendors require payment only in cash, to the courier. In other cases, only online payment on the website is suitable, with delivery to other regions.

**Reply by Yan-Aleman** [reply]
I completely agree 🔥✔️

**Comment by Saimondesigner** [comment]
Thanks for the plugin! But I understand it is not compatible with the application, fluxstore multi vendor. Then add similar code to MStore Api and fluxbuilder, so as not to install this plugin. Such functionality is necessary for wcfm users, and dokan users will also be delighted.

**Comment by Dilekmarket** [comment]
The thing is that some of the sellers are from Turkey, and the buyers are from Greece. We need that in the case of Sellers from Turkey, buyers have access to only Turkish payment gateways for payment. And for sellers from Greece, only payment by check is available (they must select it themselves, and disable all other payment methods). Tell me, please, is the ability for sellers to enable the necessary payment methods, and disable unnecessary ones, for their store - already available in the mobile application??? Because we have a shopping cart system only for one vendros, and this will not affect the choice of other vendors.


It is not very convenient to use this plugin for wcfm. The set of these functions should be available in the Sellers' personal account (enabling and disabling payment methods for buyers). Fluxbuilder should provide these features by default, on the marketplace website and in the Multi Vendor mobile application. For correct operation and display of payment methods for buyers, beneficial for vendors


And some vendors generally deliver by their own delivery, and only for cash!!

Will this problem be solved???? In the near future???
Tnx

**Reply by Yan-Aleman** [reply]
Yes, that would be cool.  We have to refuse most vendors connection to the site.  Because they want only one available type of payment for customers' orders.  Cash to the vendor's courier or only payment by card on the website.  The mobile app Flux does not support this functionality.....

**Comment by Ewachira254** [comment]
> WFCM has released a very important addon that allows Vendors to choose the preferred payment method for customers in "personal account Vendors"
> 
> The developers have added an update to the plugin. Also in the future, they will add it at the system level to the main WCFM plan.
> 
> New VERSION 1.0.2 https://drive.google.com/file/d/1qGLSRccT5qZHadJ9SGTmteirhbj7O7Mv/view?usp=sharing
> 
> Info here: https://wordpress.org/support/topic/payment-gateways-by-vendor/#post-12880625
> 
> Please make this feature compatible by default for Fluxstore Multi. It is very important and solves many problems. Since not all Vendors want to receive payment by card or cash.
> 
> Also "Payment Details" allows Vendors to attach text to the payment description. And the buyer after placing an order sees this text. Very handy when the payment method selected is bank transfer and Vendors can enter their account number in the Payment Details. The buyer will simply see it and make a transfer.

I know it is an old post .. but the update doesnt capture the vendors specific payment details... It still picks the admin payment details. The customer should be able to pay to vendors accounts . Is there a different or updated plugin.

**Comment by Ewachira254** [comment]
I know it is an old post .. but the update doesnt capture the vendors specific payment details... It still picks the admin payment details. The customer should be able to pay to vendors accounts . Is there a different or updated plugin.  I need customers to pay directly to vendors through an offiline mode like bank transfer, Cash. The vendors will be on subscription

**Reply by Yan-Aleman** [reply]
at least to have the ability to include specific payment methods for vendors, this is already a big plus. Because some want to work with cash payments upon delivery, in the hands of the courier. And others are satisfied only with online payments on the site. Each vendor has its own request. In general, we are waiting for a solution to this problem in the FluxStore MultiVendor application...

**Reply by Ewachira254** [reply]
where can I get the updated original plugin where at the checkout point the customer gets the vendors payment mode and payment details instead of the payment details to admin. I want the customers to pay directly  to the vendors. I will appreciate.

**Reply by Yan-Aleman** [reply]
> where can I get the updated original plugin where at the checkout point the customer gets the vendors payment mode and payment details instead of the payment details to admin. I want the customers to pay directly  to the vendors. I will appreciate.
> 

Unfortunately, this plugin can only make different payment methods available for vendors. Enable or disable. Other plugins have not yet been found. This is a rather acute problem. Maybe the flux team will release an add-on and supplement the existing functionality with the ability to pay directly to suppliers. We really count on it.

**Reply by Ewachira254** [reply]
![WCFM Plugin](https://github.com/user-attachments/assets/9e49c7d6-b834-47b7-8a61-5665d159fa61)
This is the plugin update I am referring to. I just want where the vendor puts his payments mode and instructions and the customers sees them details when checking out. I dont need the fluxstore one.. I need the one for general WCFM.
